### ISSUE TITLE
[CPT] I can connect to a remote runtime

### ISSUE THREAD

**Issue by saig0** [camunda/camunda#25708]
## Description

Instead of running the process tests with a local runtime inside Testcontainers, I want to run the tests with a remote runtime to perform e2e tests. 

In this mode, CPT doesn't manage the runtime but connects to an existing runtime. 

## Tasks
- Provide an option to connect to a remote runtime
- Disable the start of a local runtime 

Requires: CPT uses only public APIs (or available in SaaS + Self-managed)

:lock: Related [Slack discussion](https://camunda.slack.com/archives/C06VC96GMNE/p1733240958770759).

**Comment by npepinpe** [comment]
Quick question, will it also purge my remote runtime between tests? I think so, but just want to confirm.

**Comment by saig0** [comment]
@npepinpe yes, CPT will try to purge the remote runtime between the test cases. I tested the behavior with C8run. 👍
